server/app.py

```diff
@@ -88,18 +92,36 @@ async def _calculate_fps_loop(self):
                 current_time = time.monotonic()
                 if self._last_fps_calculation_time is not None:
                     time_diff = current_time - self._last_fps_calculation_time
-                    self._fps = self._fps_interval_frame_count / time_diff
+                    self._fps = (
+                        self._fps_interval_frame_count / time_diff
```

Comment:
Moved over to here to prevent duplicate calculations and makes it available for the prometheus metrics.